dll/win32/winmm/midimap/midimap.c

```diff
@@ -239,6 +239,11 @@ static BOOL	MIDIMAP_LoadSettings(MIDIMAPDATA* mom)
 	    {
 		ret = MIDIMAP_LoadSettingsDefault(mom, buffer);
 	    }
+        else if (!RegQueryValueExW(hKey, L"szPname", 0, &type, (void*)buffer, &size) && *buffer)
```

Comment:
Indentation fix please.

Reply:
Wine code have mixed tabs-spaces indentation, not sure how to fix it properly.

Here I just used default 4-space indentation.

Reply:
@HBelusca ok I've duplicated whitespace from bottom code block, should look better now.